contrib/config/vault/docker/README.md

```diff
@@ -211,15 +221,15 @@ curl --silent \
   http://$VAULT_ADDRESS/v1/secret/data/dgraph/alpha | jq
 ```
 
-**NOTE**: When updating K/V Version 2 secrets, be sure to increment the `options.cas` value to increase the version.  For example, if updating the `enc_key` value to 32-bits, you would update `./vault/payload_alpha.secrests.json` to look like the following:
+**NOTE**: When updating K/V Version 2 secrets, be sure to increment the `options.cas` value to increase the version.  For example, if updating the `enc_key` value to 32-bits, you would update `./vault/payload_alpha_secrets.json` to look like the following:
 ```json
 {
   "options": {
     "cas": 1
   },
   "data": {
     "enc_key": "12345678901234567890123456789012",
-    "hmac_secret_file": "12345678901234567890123456789012"
+    "hmac_secret": "12345678901234567890123456789012"
```

Comment:
are we sure of the key name change? In dgraph I still see using hmac_secret_file.